packages/core/content-type-builder/admin/src/components/FormModal/attributes/types.js

```diff
@@ -172,7 +172,13 @@ const types = {
           name: 'doesNotStartWithANumber',
           message: getTrad('error.validation.enum-number'),
           test: values => !values.some(startsWithANumber),
+        })
+        .test({
+          name: 'matchesRegex',
+          message: getTrad('error.validation.match-regex'),
```

Comment:
To be more consistent with the other error names, could you please prefix the message:

```suggestion
          message: getTrad('error.validation.enum-match-regex'),
```